src/amo/components/WrongPlatformWarning/index.js

```diff
@@ -54,7 +54,7 @@ export class WrongPlatformWarningBase extends React.Component<InternalProps> {
   static defaultProps = {
     _correctedLocationForPlatform: correctedLocationForPlatform,
     _getClientCompatibility: getClientCompatibility,
-    _isFenix: isFenix,
+    _isFenix: isFirefoxForAndroid,
```

Comment:
This is weird.

Reply:
Could you update the injected prop name?

Reply:
Yeah, I'm not sure what happened here. Probably an artifact of how I tried to extract some changes from a larger patch. I'll make sure this is fixed in the big patch.